src/mlpack/methods/ann/layer/atrous_convolution_impl.hpp

```diff
@@ -63,45 +63,23 @@ AtrousConvolution<
     const size_t inputHeight,
     const size_t dilationWidth,
     const size_t dilationHeight,
-    const std::string paddingType) :
-    inSize(inSize),
-    outSize(outSize),
-    kernelWidth(kernelWidth),
-    kernelHeight(kernelHeight),
-    strideWidth(strideWidth),
-    strideHeight(strideHeight),
-    inputWidth(inputWidth),
-    inputHeight(inputHeight),
-    outputWidth(0),
```

Comment:
padW and padH weren't initialized here, I have added that I hope that's ok.
Thanks.